app/src/main/java/org/fossasia/openevent/app/contract/presenter/LoginPresenter.java

```diff
@@ -0,0 +1,15 @@
+package org.fossasia.openevent.app.contract.presenter;
+
+import org.fossasia.openevent.app.contract.view.LoginView;
+
+public interface LoginPresenter {
+
+    void start();
+
+    void stop();
+
+    void login(String email, String password);
+
+    LoginView getView();
```

Comment:
@iamareebjamal This is not required. as we can pass the view to presenter through the constructor from the view itself.